pkg/render/intrusion_detection.go

```diff
@@ -631,13 +634,30 @@ func (c *intrusionDetectionComponent) intrusionDetectionControllerContainer() co
 			Name:  "FIPS_MODE_ENABLED",
 			Value: operatorv1.IsFIPSModeEnabledString(c.cfg.Installation.FIPSMode),
 		},
+		{
+			Name:  "LINSEED_URL",
+			Value: "https://tigera-linseed.tigera-elasticsearch.svc",
```

Comment:
Do we not have a constant for this URL? We probably should have one since it will be used everywhere.